### DISCUSSION TITLE
申请退款。购买一年专业版，三天后不管是手机版还是专业版都不能连接外网。

### DISCUSSION THREAD

**Discussion by MuliousZhang** [getlantern/lantern-forum-cn#916] [General]
[Mike777Khoo](https://github.com/Mike777Khoo)请帮我退款，尽快。等蓝灯稳定了我再考虑继续使用。
邮箱zjz_mulious@163.com
付款凭证
![Screenshot_20241001_195351_com eg android AlipayG](https://github.com/user-attachments/assets/54062221-ec56-49ae-97cc-70addf505078)

**Comment by MuliousZhang** [comment]
@Mike777Khoo请帮我退款，尽快。

**Comment by MaxMKMax** [comment]
@MuliousZhang 您好. 好的, 我帮您提交退款. 您的情况比较具挑战性, 那只能退款了.

**Reply by MuliousZhang** [reply]
多久后能退款到账？

**Comment by MuliousZhang** [comment]
@Mike777Khoo 每日打卡，直到退款为止。

**Reply by MuliousZhang** [reply]
继续打卡。蓝灯仍然是偶尔能连接几秒钟到几分钟的状态而且也没有给我退款。

**Comment by MuliousZhang** [comment]
依然是偶尔能连接几秒钟到几分钟，不管手机还是电脑都用不了。而且还是没有退款。

**Comment by MuliousZhang** [comment]
又是不能连接也没有退款的一天

**Comment by MuliousZhang** [comment]
继续打卡，依然没有退款。

**Comment by MaxMKMax** [comment]
@MuliousZhang 您好. 我再度帮您追问一下啊. 稍安.

**Comment by greySkiesMarsh** [comment]
Hi @MuliousZhang, I've been investigating this issue and need some clarification to get it sorted out for you. 

Did you purchase your Pro subscription through the app itself, or did you order a code through a 3rd party seller? 

I ask because I was not able to track down your transaction through any of our internal payment portals, but I want to assist you as best I can.

**Reply by MuliousZhang** [reply]
through the app purchased my Pro subscription on the app, paid with Alipay, and got the screenshot of the payment record as shown above. itself
